### PR TITLE
Add allowed values for the nature-of-agreement property

### PR DESCRIPTION
**Pull request by DimitriZhurkin**
In the **Leveraged FedRAMP-authorized Services** section, add allowed values for the `nature-of-agreement` property.

### CODE REVIEW

**Comment by DimitriZhurkin** [comment]
Related PR: [#920](https://github.com/GSA/fedramp-automation/pull/920)

**Review by wandmagic — APPROVED** [review]

**Review by Gabeblis — CHANGES_REQUESTED** [review]
Just a couple suggestions for consistency.

File: content/documentation/ssp/4-ssp-template-to-oscal-mapping.md [hunk 1]
```diff
@@ -979,6 +980,20 @@ The title field must match an existing [FedRAMP authorized Cloud_Service_Provide
 
 A leveraged-system-identifier property must be provided within each leveraged-authorization field.  The value of this property must be from the same Cloud Service Provider as identified in the title field.
 
+{{</callout>}}
+
+{{<callout>}}
+
+##### FedRAMP Allowed Values
```
[the remark this answers is not in the document]
Comment (DimitriZhurkin) [reply content/documentation/ssp/4-ssp-template-to-oscal-mapping.md]: Fixed.

File: content/documentation/ssp/4-ssp-template-to-oscal-mapping.md [hunk 2]
```diff
@@ -979,6 +980,20 @@ The title field must match an existing [FedRAMP authorized Cloud_Service_Provide
...
+The `nature-of-agreement` property identifies the appropriate type of agreement between the documented system and its leveraged authorizations documented in an SSP.
+
```
[the remark this answers is not in the document]
Comment (DimitriZhurkin) [reply content/documentation/ssp/4-ssp-template-to-oscal-mapping.md]: Moved the explanation up.

**Review by Gabeblis — APPROVED** [review]

**Review by aj-stein-gsa — COMMENTED** [review]
We are going to start using callouts for allowed-value enumeration sets?

**Comment by aj-stein-gsa** [comment]
> > We are going to start using callouts for allowed-value enumeration sets?
> 
> 
> 
> We have been using callouts for allowed-value enumeration sets up to this point. Should this be changed? 

That's ok I just didn't quickly find an example.

**Review by aj-stein-gsa — APPROVED** [review]

**Pull request merged**